src/scheduler/fifo.c

```diff
@@ -2054,14 +2054,9 @@ add_job_to_calendar(int pbs_sd, status *policy, server_info *sinfo,
 		schdlog(PBSEVENT_DEBUG, PBS_EVENTCLASS_JOB, LOG_DEBUG,
 			bjob->name, log_buf);
 	} else if (start_time == 0) {
-		/* In the case where start_time = 0, we don't want mark the job as
```

Comment:
why did you remove this comment? is this information incorrect? is yes then we should probably modify the comment to mention the correct reason why we aren't marking the job as can_never_run..

Reply:
The comment was obsolete.  The old message that was printed said the job would never run.  This confused people since they'd see a message saying the job would never run, and then later in the day it'd run.  The problem could have been that a bunch of nodes were down making it impossible to run the job in the system at that time.  Once the nodes came back online, the job ran.  I changed this message to say we had an error adding the job to the calendar.  The comment referred to the previous log message.